control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2978,6 +3028,39 @@ components:
       required:
         - spec
         - state
+    Snapshot:
+      description: |-
+        Snapshot Information
+      type: object
+      properties:
+        uuid:
+          $ref: '#/components/schemas/SnapshotId'
+        name:
+          description: Name of the snapshot
```

Comment:
Do we end these with `.` ? Others below have it, though I don't know if we have such practice already or not.

Reply:
I think we should follow this. To be consistent. I will add wherever needed in my changes.